.github/workflows/release.yml

```diff
@@ -12,6 +12,12 @@ jobs:
       contents: write
     steps:
     - uses: actions/checkout@v3
+    - name: set tag variable in atlas
+      run: |
+        # Sets _ATLAS_VERSION to the latest tag
+        CURRENT_TAG=$(git describe --exact-match --tags)
+        sed -i -e "s/_ATLAS_VERSION=.*/_ATLAS_VERSION=\"${CURRENT_TAG}\"  # Tagged by release.yml/" atlas
     - uses: ncipollo/release-action@v1
       with:
+        generateReleaseNotes: true
```

Comment:
I prefer to have the default GitHub generated release notes because I find it useful.